crm_profiling/__manifest__.py

```diff
@@ -19,7 +19,7 @@
     """,
     "author": "OpenERP SA",
     "website": "https://github.com/OCA/server-tools",
-    "depends": ["base", "crm", "sale"],
+    "depends": ["account", "contacts", "queue_job_batch"],
```

Comment:
@nikunjantala 

The module "queue_job_batch", where are you pulling it from? as in OCA repository, it is not available for v15

https://github.com/OCA/queue/tree/15.0

Reply:
https://github.com/OCA/queue/pull/500